modules/images/webp-uploads/load.php

```diff
@@ -39,7 +39,7 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 		return $metadata;
 	}
 
-	$file = get_attached_file( $attachment_id, true );
+	$file = wp_get_original_image_path( $attachment_id, true );
```

Comment:
Mostly to avoid the need to pass in filters when accessing the file. As in case of wanted to filter here there's always `wp_get_original_image_path` let me know if you think that makes sense.

Reply:
> Mostly to avoid the need to pass in filters when accessing the file.

But why? I'd say we can let the filters run as usual here.

Reply:
Well to avoid the need for an additional call to a filter in a place where we are not really required due we need to access to the actual image file. 

I see the benefit in keeping the filters to maximize compatibility tho.